bridge/client/app/dashboard-legacy/dashboard-legacy.component.spec.ts

```diff
@@ -12,7 +13,7 @@ describe('DashboardLegacyComponent', () => {
   beforeEach(async () => {
     await TestBed.configureTestingModule({
       imports: [HttpClientTestingModule],
-      providers: [],
+      providers: [{ provide: POLLING_INTERVAL_MILLIS, useValue: 0 }],
```

Comment:
We don't need this provider as we call the constructor of this component on our own

Reply:
Fixed